CODEOWNERS

```diff
@@ -218,7 +218,6 @@
 /lib/posix/                               @pfalcon
 /kernel/device.c                          @andrewboie @andyross @nashif
 /kernel/idle.c                            @andrewboie @andyross @nashif
-/samples/basic/servo_motor/*microbit*     @jhe
```

Comment:
Looks like a stray change that doesn't belong in this commit

Reply:
I have split the commit into two, to make the removal of the "clock-prescaler" property and the above entry from CODEOWNERS more visible and clear.